ex1/dates_times.py

```diff
@@ -26,19 +26,73 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+from datetime import date, time, datetime, timedelta
+
 '''
 Exercises for using the datetime and the calendar module
 '''
 
 # Define a function named last_of_month that takes an argument dt of type date
 # and returns a date object representing the last day of the month dt was in.
+def last_of_month(dt):
+	"""
+	Parameters
+	----------
+	dt : datetime
+
+	Returns
+	-------
+	datetime
+		last day of the month
+    	"""
+	if dt.month == 12:
+		return dt.replace(day=31)
+	return dt.replace(month=dt.month+1, day=1) - timedelta(days=1)
```

Comment:
:smiley: Smart but unnecessary. See the [calendar.monthrange](https://docs.python.org/3.4/library/calendar.html#calendar.monthrange) function or #20